src/logic/safe/store/middleware/safeStorage.ts

```diff
@@ -28,8 +28,8 @@ export const safeStorageMiddleware = (store: Store) => (
 
     switch (action.type) {
       case SET_DEFAULT_SAFE: {
-        if (action.payload) {
-          saveDefaultSafe(action.payload as string)
+        if (typeof action.payload === 'string') {
```

Comment:
What was passed there? 👀

Reply:
an empty string

Reply:
I think this check is not really needed because any problem like that should be caught by static typing, but I'm not ready to take more risks rn xD

Reply:
The safer the better 👍